### DISCUSSION TITLE
Security Onion 2.3.2 - Installation Kernel Panic

### DISCUSSION THREAD

**Discussion by tcritch05** [Security-Onion-Solutions/securityonion#1923] [General] [answered]
Has anyone else ran into a kernel panic when trying to install 2.3.2? I'm using a Micro Dell Optiplex 5070 as my sensor which ran Security Onion 16.04 without any issues for several months. At first I thought it might be related to a bad .iso but I was able to verify it using the documentation. 

https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md

Same result using different bootable USB which has been created using Balena Etcher.

To test the hardware, I installed the latest version of CentOS7-2009 with my original USB and Etcher. No issues getting that installed. 

I've updated my BIOS the latest and looked through all the BIOS settings that might be causing installation issues as well as trying to install in both UEFI and Legacy. Not sure where to go from here... any ideas? 

![PXL_20201115_215222902 MP](https://user-images.githubusercontent.com/34383711/99199316-de509280-2763-11eb-9bc3-17143f3fc6c5.jpg)

**Comment by TOoSmOotH** [comment, marked as answer]
If CentOS 7 is installing I would just use that and do a network based install. Not sure why it is crashing.
